server/src/graphql/resolvers/contentTextWithType.js

```diff
@@ -0,0 +1,68 @@
+const { db } = require('../../db/models');
+
+module.exports = {
+  async capabilities(parent) {
+    const capabilitiesArray = await db.contentText.findAll({
+      where: { type: 'b063378f-5ffe-4bba-99a8-e4c5134fff23' },
```

Comment:
Hard coding primary key values doesn't feel like a good practice. If you can't find a better way reach out to a mentor and ask for advice

Reply:
I can't figure out a solution I like. Keep it as it is